f2/src/components/formik/fileUpload/index.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react'
-import { FormFile, Row } from 'react-bootstrap'
+import { FormFile } from 'react-bootstrap'
 import { UploadButton } from '../button'
 import { acceptableExtensions } from '../../../utils/acceptableFiles'
+import { FormRow } from '../row'
 
 export const FileUpload = ({ field, form, onChange, ...props }) => {
   return (
-    <Row className="form-row">
+    <FormRow>
```

Comment:
Would FormRow without any attributes just adds following CSS?
 margin-bottom: 1rem;
  width: 100%;

Reply:
Yes, this would be the default style. If margin-bottom or width are passed it will override this default.